commonui/src/main/java/com/hyperwallet/android/ui/common/insight/HyperwalletInsight.java

```diff
@@ -86,7 +86,7 @@ public static void setInstance(HyperwalletInsight hyperwalletInsight) {
      */
     public void initialize(@NonNull final Context context, @NonNull final Configuration configuration) {
         Insight.initialize(context, configuration.getEnvironment(), configuration.getProgramToken(), SDK_VERSION,
-                configuration.getInsightApiUrl(), configuration.getUserToken());
+                configuration.getInsightApiUri(), configuration.getUserToken());
```

Comment:
FYI @peter-joseph  worked on this PR  https://github.com/hyperwallet/hyperwallet-android-ui-sdk/pull/139/files

Reply:
Lets merge this first as it is failing the build for some of the other PRs